### PR TITLE
Smart contract updates

### PR DESCRIPTION
**Pull request by mikec**
some code DRYing and additional tests for OracleManagerApp.sol ... in prep for the add/remove data feeds UI

### CODE REVIEW

File: contracts/OracleManagerApp.sol [hunk 1]
```diff
@@ -15,25 +15,22 @@ contract OracleManagerApp is AragonApp {
...
-  function initialize(DataFeedOracleBase[] memory dataFeeds, ITidbitDataFeedOracle _medianDataFeed)
-    public
-    onlyInit
+  function initialize(DataFeedOracleBase[] dataFeeds, ITidbitDataFeedOracle _medianDataFeed)
+    external
   {
     initialized();
-
-    approvedDataFeedsLength = dataFeeds.length;
+
```
Comment (emaG3m) [inline contracts/OracleManagerApp.sol]: xtra space

File: contracts/OracleManagerApp.sol [hunk 2]
```diff
@@ -15,25 +15,22 @@ contract OracleManagerApp is AragonApp {
   ITidbitDataFeedOracle public medianDataFeed;  // tidbit MedianDataFeedOracle to record median data throughout time
   uint public approvedDataFeedsLength;   // number of approvedDataFeeds
   mapping(address => bool) public approvedDataFeeds; // dataFeeds approved to be medianized
-  mapping(address => bool) dataFeedAlreadyRecorded; // transitory data structure useful only during function call recordDataMedian
```
Comment (emaG3m) [inline contracts/OracleManagerApp.sol]: I still like my wording better. more specific takes less seconds to understand when going through the code

Comment (mikec) [reply contracts/OracleManagerApp.sol]: can we call it `dataFeedIsSet`?

File: test/OracleManagerApp.test.js [hunk 3]
```diff
@@ -54,6 +54,18 @@ contract('OracleManagerApp', (accounts) => {
   })
 
   describe('initialize()', () => {
+    it('initializes the contract', async () => {
+      await oracleManagerApp.initialize([dataFeed1.address, dataFeed2.address], medianDataFeed.address)
+      expect(await oracleManagerApp.hasInitialized()).to.equal(true)
```
Comment (emaG3m) [inline test/OracleManagerApp.test.js]: 💯

File: test/OracleManagerApp.test.js [hunk 4]
```diff
@@ -54,6 +54,18 @@ contract('OracleManagerApp', (accounts) => {
...
+    })
+
+    it('reverts if called a second time', async () => {
```
Comment (emaG3m) [inline test/OracleManagerApp.test.js]: 💯

**Comment by emaG3m** [comment]
👍

**Pull request merged**